app/controllers/refusal_advice_controller.rb

```diff
@@ -1,10 +1,47 @@
 class RefusalAdviceController < ApplicationController
   def create
-    @params = parsed_refusal_advice_params # TODO: move into log event method
```

Comment:
The approach of logging a separate event and passing through a separate controller feels like it's adding quite a bit of complexity. I don't have fully-formed thoughts on this yet though.

There are two things rattling around in my head.

1. We already log `followup_sent`, which is the main "event" that's happening. I can appreciate the technical reasons for why a `refusal_advice_wizard` has been added, but it doesn't feel like we're logging something that's particularly actionable or driven by user intent. IDK, I could be convinced otherwise I guess. My thinking though, was that we'd temporarily store some data from the refusal advice form and then record that in the `params_yaml` of the subsequent `followup_sent` event. 
2. Maybe more in my mind as feeling weird is that some installs won't configure refusal advice, so they'll need to be concerned about where to send the user from their implementation of `/help/unhappy`. Again, I'm not totally against what we have here – this is just thinking out loud. It feels like this couples followups with refusal advice more than it should. We do want to be able to show snippets based on refusal advice though, and this makes it easier for reusers to configure in that they don't have to worry about matching up the `response_template` key with tags in the database?

I'm wondering whether there's a middle ground where we can reduce the complexity of this action, but still with the gains of reduced configuration.

Reply:
I was kinda viewing this as an event which happend to the info request. We talked about recording usage/results of the wizard which this does and allows us to anaylise usage at a later date.  

I don't think its too complex, all `parsed_refusal_advice_params` is doing is turning `"true"`/`"false"` into booleans to be stored in the database. Would creating a seporate refusal advice answers model be better - It could do this parsing for us then?

Reply:
For now I've continued with creating `InfoRequestEvent` but am open to changing now or in the future.

Reply:
> I was kinda viewing this as an event which happend to the info request. We talked about recording usage/results of the wizard which this does and allows us to anaylise usage at a later date.

Yeah, I'm not against per-say, just made me pause for thought.

> I don't think its too complex, all `parsed_refusal_advice_params` is doing is turning `"true"`/`"false"` into booleans to be stored in the database. Would creating a seporate refusal advice answers model be better - It could do this parsing for us then?

A PORO like `RefusalAdvice::WizardAnswers.new(params).to_hash` might be nice, but not essential.

Reply:
> 2\. Maybe more in my mind as feeling weird is that some installs won't configure refusal advice, so they'll need to be concerned about where to send the user from their implementation of `/help/unhappy`. Again, I'm not totally against what we have here – this is just thinking out loud. It feels like this couples followups with refusal advice more than it should. We do want to be able to show snippets based on refusal advice though, and this makes it easier for reusers to configure in that they don't have to worry about matching up the `response_template` key with tags in the database?

This is maybe my main concern here. How will this work?